backend/endpoints/trainspace_endpoints.py

```diff
@@ -39,8 +39,8 @@ def create_trainspace():
       - 400: Error in creating trainspace
     """
     try:
-        request_args = json.loads(request.args)
-        uid = request_args["user"]["uid"]
+        # request_args = json.loads(request.args)
```

Comment:
is this commented out code needed @archishmagoli

Reply:
No, I can remove it if the fix works.